### PR TITLE
Fully tested update

### PR DESCRIPTION
**Pull request by sean-m-sullivan**
# What does this PR do?
Fully tested update with more tests added, two new roles, this should be on the verge of being release ready, needs review.

### CODE REVIEW

**Review by Tompage1994 — APPROVED** [review]
LGTM

File: roles/ansible_config/tests/vars/config.yml [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+    keypairs:
+      - key: ignore_certs
+        value: "{{ not (aap_validate_certs | bool) }}"
+      - key: server_list
+        value: "{{ automation_hub_list | map(attribute='name') | join(',') }}"
 
 automation_hub_list:
   - name: automation_hub
-    url: https://cloud.redhat.com/api/automation-hub/
+    url: "{{ah_host}}/api/automation-hub/content/0000001-synclist/"
```
Comment (djdanielsson) [inline roles/ansible_config/tests/vars/config.yml]: spacing?

Comment (sean-m-sullivan) [reply roles/ansible_config/tests/vars/config.yml]: fixed

**Review by djdanielsson — APPROVED** [review]
looks good now

**Pull request merged**